notebooks/examples/analog_sensitivity_LeNet5.ipynb

```diff
@@ -0,0 +1,1483 @@
3JBdZsctcxPK5G7scjvrlV0ukFWac3U3gVbdqIURKy3QWoLXBVrnnntuufPOO2c8N9hgg3LAAQeUHXfccZnGf/VXf1W+9a1vzfx91VVXLRtvvHFZd911y8MPPzz60vif/vSno/+95NU90XXMMceU3/3d322hd1NzDAKt4baqb7gwX4MZbvmc7muXb7lfpQvH3I9dZsctcxPK5G7scjvrlV0ukFWac3U3gVbdqIURKz3Q6hC7p6c++9nPjp6kuu+++2Zct9lmm/K6172ubLHFFo+xvv/++0dPWnUfJ5zNq3uS681vfnPZfvvtZzPcmDEKCLTGiDnmt+obLgi0sobMV7dOq++cy8T7V7lwzA3ZZXbcMjehTO7GLrezXtnlAlmlOVd3E2jVjVoYMYhAawnk3XffXc4777yZXyPs/r178qr7Avd99923rL322o8x7z5+eMUVV5Sbb755qf3oan7/939/9GuKG220UQs9m7pjEGgNt2V9w4X5Gsxwy+d0X7t8y/0qXTjmfuwyO26Zm1Amd2OX21mv7HKBrNKcq7sJtOpGLYwYVKC1BPR73/ve6Pu1brvtthnjddZZp+y///7lD//wD5fqfu+995Yf/vCHZdGiRaMnvtZaa62yySablM0226ysssoqLfRqao9BoDXc1vUNFwRaWW/nq1un1XfOZeL9q1w45obsMjtumZtQJndjl9tZr+xygazSnKu7CbTqRi2MGGSg1cF2333VfaH7pz71qfKrX/1qxvopT3nK6GOIfqVweqafQGu4veobLszXYIZbPqf72uVb7lfpwjH3Y5fZccvchDK5G7vcznpllwtkleZc3U2gVTdqYcRgA60luF2YddFFF5UvfvGL5aGHHhr9c/fE1S677FJe/epXl+7JLa9hCwi0htufvuGCQCvr7Xx167T6zrlMvH+VC8fckF1mxy1zE8rkbuxyO+uVXS6QVZpzdTeBVt2ohRGDD7SWIHcfJzz77LNL93HEJa/u+7H22Wefsttuu42+a8trmAICrWH2ZRzhwnwNZvqGMvPVbRxzbmWtJheOuTy7zI5b5iaUyd3Y5XbWK7tcIKs05+puAq26UQsjpibQWoJ9/fXXlwsuuKDcddddM/7dryC+9rWvLb/7u7/bQk+aOwaB1nBbKpjJesMtcxNo5W7TXOmiO+set8xNKJO7scvtrFd2uUBWac7V3QRadaMWRkxdoNWh/+Y3vyn/63/9r3L55ZePvgB+yWuHHXYoBxxwQNlwww1b6E0zxyDQGm4rBTNZb7hlbgKt3G2aK110Z93jlrkJZXI3drmd9couF8gqzbm6m0CrbtTCiKkMtJbA33nnneWcc84p3/zmN2d6sfrqq5eXvexlZc899yzdJPZa+QICrZXfg2XtgWAm6w23zE2glbtNc6WL7qx73DI3oUzuxi63s17Z5QJZpTlXdxNo1Y1aGDGYQOu2224r1157bbn55ptLF4Dce++9o0Cq+9L3Jz7xiWXbbbctO+20U1lzzTUf4/6P//iP5ZOf/GT58Y9/PPO3TTbZpBx88MHlec97Xgt9mupjEGgNt32Cmaw33DI3gVbuNs2VLrqz7nHL3IQyuRu73M56ZZcLZJXmXN1NoFU3amHESg+0ul8xPPPMM8tXvvKVqufChQtH35XVBVu//eo+evh3f/d35dJLLx19JHHJa7vtthsFW5tuumn1/Q2YjIBAazKu43hXwUymyC1zE2jlbtNc6aI76x63zE0ok7uxy+2sV3a5QFZpztXdBFp1oxZGrNRAqwuzjjvuuHL77bevkOVBBx1UXvrSly61pvuy+PPOO6/ccMMNM3/vJvPuu+9eXv7yl5c11lhjhbZlcH8BgVZ/w0m9g2Amk+WWuQm0crdprnTRnXWPW+YmlMnd2OV21iu7XCCrNOfqbgKtulELI1ZqoPU//+f/LN2vFi55bbzxxuWP//iPy5Zbbjn6YveHH3549PHDr3/96+Waa64pDzzwwGjoqquuWj74wQ+W7tcNl/X6zne+U84+++xHhWUbbLDB6Evjd9xxxxZ6NzXHINAabqsEM1lvuGVuAq3cbZorXXRn3eOWuQllcjd2uZ31yi4XyCrNubqbQKtu1MKIlRZodU9SvelNb5ox7IKs7qOBXVi1tNePfvSj8u53v7v84he/GP35D//wD8thhx223B489NBD5corryx/8zd/U7qnwZa8ulBrjz32aKF/U3EMAq3htkkwk/WGW+Ym0MrdprnSRXfWPW6Zm1Amd2OX21mv7HKBrNKcq7sJtOpGLYxYaYHWddddV0499dSRYbcgTz755GWGWUugv/SlL5XTTz999J/d01zdE16zeXVh1l//9V+Xq6++evTU1z777FNe8YpXzKbUmDEICLTGgDihtxDMZLDcMjeBVu42zZUuurPuccvchDK5G7vcznpllwtkleZc3U2gVTdqYcRKC7Quu+yycuGFF44Md9hhh/K2t72t6vnv//7v5YgjjhiNW2211cr5559frXnkgFtvvXX0McTnPOc5Aq0Vkus3WKDVz2+S1YKZTJdb5ibQyt2mudJFd9Y9bpmbUCZ3Y5fbWa/scoGs0pyruwm06kYtjFhpgdYVV1wxCpe6VxcwHXPMMVXP73//++XYY48djXvc4x5XzjrrrGrN0gZ0H3fsvk/La24EBFpz45xsRTCTqJXCLXMTaOVu01zpojvrHrfMTSiTu7HL7axXdrlAVmnO1d0EWnWjFkastEDrn//5n8t73/vekWE32boved9ss82Wa9p9RLH7qGL3evrTn17e8573tNCD5o9BoDXcFgtmst5wy9wEWrnbNFe66M66xy1zE8rkbuxyO+uVXS6QVZpzdTeBVt2ohRErLdBavHhxefOb3zzzJe/rrbdeOfTQQ8sLXvCCx7guWrSoXHTRReWLX/zizN+6L5D/kz/5kxZ60PwxCLSG22LBTNYbbpmbQCt3m+ZKF91Z97hlbkKZ3I1dbme9sssFskpzru4m0Kob
```

Comment:
Best to mention here that retraining is done in software using the non-idealities in the forward pass but perfect backward pass and update. You can also link to the explanation in the docs: https://aihwkit.readthedocs.io/en/latest/hwa_training.html